src/auth.ts

```diff
@@ -102,6 +102,10 @@ export async function auth(
         ...authentication,
       };
 
+      // if (state.onTokenCreated) {
+      state.onTokenCreated(state.authentication, options.type);
```

Comment:
Passing `options.type` here would be enough info? @ofhouse

Reply:
I'd prefer to pass an `{ type }` object, it will make it future proof in case we want to pass more options